docs/source/index.rst

```diff
@@ -58,15 +58,15 @@ Stable Packages
 +--------------------------------------------+--------------------+--------+
 | Daml.Finance.Claims                        | 1.0.1              | Stable |
 +--------------------------------------------+--------------------+--------+
-| Daml.Finance.Data                          | 1.0.1              | Stable |
+| Daml.Finance.Data                          | 2.0.0              | Stable |
 +--------------------------------------------+--------------------+--------+
 | Daml.Finance.Holding                       | 1.0.2              | Stable |
 +--------------------------------------------+--------------------+--------+
 | Daml.Finance.Instrument.Generic            | 1.0.1              | Stable |
 +--------------------------------------------+--------------------+--------+
 | Daml.Finance.Instrument.Token              | 1.0.1              | Stable |
 +--------------------------------------------+--------------------+--------+
-| Daml.Finance.Interface.Data                | 1.0.1              | Stable |
+| Daml.Finance.Interface.Data                | 2.0.0              | Stable |
```

Comment:
Not for this PR, but I think we should review our definition of the public API to exclude interface methods --> only the choices are part of the API